python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -15,7 +15,7 @@
 logger = getLogger()
 
 _reg_id_str = '_registry_id'
-_id_str = 'id'
+_id_str = '_id'
```

Comment:
Ok, it looks like going back to 6.0.44, `_registry_id` indeed was used but the *only* thing it was used for was a small amount of caching to make `Registry.find` a little faster. I'm happy to keep them for now but we should consider in the future work towards decoupling them.